opensearch-operator/controllers/cluster_test.go

```diff
@@ -84,7 +84,22 @@ var _ = Describe("Cluster Reconciler", func() {
 		It("should apply the cluster instance successfully", func() {
 			Expect(k8sClient.Create(context.Background(), &OpensearchCluster)).Should(Succeed())
 		})
-
+		//	Sm := &prometheus.ServiceMonitor{}
```

Comment:
Why is this test commented out?
Also: There is no specific unittest for the monitoring feature.

Reply:
Sorry, changed and added more tests